### ISSUE TITLE
508C compliance

### ISSUE THREAD

**Issue by walkerjeffd** [HWR-llc/ett#57]
Ok, here's the fun part!

For the most part, we should be ok with the 508C (accessibility) because bootstrap is generally compatible (if we were writing our own custom CSS, that would likely not be the case).

Read this: https://getbootstrap.com/docs/4.0/getting-started/accessibility/

### Color Contrast

First, test color contrast of each color combo (mainly buttons, but generally anywhere that we do not have black text on white background or vice versa).

I like this tool: https://accessibleweb.com/color-contrast-checker/

Use the eye dropper to select the background of each button or element. I can already tell you our "Start Exploring" button fails.

![image](https://user-images.githubusercontent.com/981144/170275844-825162a3-7690-4073-907c-237ef85297d0.png)

Will need to adjust to a different background color that achieves compliance, or use black text:

![image](https://user-images.githubusercontent.com/981144/170276064-1b2f6ba6-30f2-40dc-90f1-5ed7a3c14f0b.png)

### Image Alt Text

Every image needs to have alt text for screen readers:

```html
<img src="/img/landing_page.250abb15.png" style="width: 35%;" alt="Photos of coastal habitat types and ETT logo">
```

I would search for each `<img>` tag and add an `alt` attribute to it.

### Accessibility Scan
 
Lastly, we should run an accessibility scanner. These are browser plug ins that can scan a page and tell you if anything is not 508C compliant. I use this one: https://www.siteimprove.com/integrations/browser-extensions/

As you can see, it already flags the lack of an alt attribute on the homepage image (that's why I listed those first before doing the scan):

![image](https://user-images.githubusercontent.com/981144/170277283-c7146ecf-faee-4c28-aa71-c84a2a26a1f3.png)

You'll need to run this on each "page" separately as it does not crawl the entire site.

Don't worry about getting a perfect score, as there are usually false-positives (i.e. errors we can safely ignore). Focus on things that you can understand and figure out how to fix (some of them are quite cryptic, don't stress over those too much). If you try a different scanner, you'll likely get different results. They also don't work as well on complex data viz applications, they're more consistent for a blog, say. So keep in mind, there's no objective way to do this really. Just try to fix as many things as you can.

Once you're done, let me know and I'll take a look and see if I spot anything else that can or should be fixed.
